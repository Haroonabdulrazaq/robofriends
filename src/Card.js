import React from 'react';

const Card = (props) => {
  const { id, name, email } = props.robot
  return (
    <div key={id} className='tc grow bg-light-green br3 pa3 dib ma2 bw2 shadow-5'>
      <img alt={name} src={`https://robohash.org/${id}?100x100`} />
      <h1>{name}</h1>
      <h3>{email}</h3>
    </div>
  )
}

export default Card
