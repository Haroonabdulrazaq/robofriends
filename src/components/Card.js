import React from 'react';

const Card = (props) => {
  const { id, name, email } = props.robot
  return (
    <div key={id} className='tc grow bg-light-green br3 pa3 dib ma2 bw2 shadow-5'>
      <img alt={name} src={`https://robohash.org/${id}?size=100x100`} />
      <h2>{name}</h2>
      <h3>{email}</h3>
    </div>
  )
}

export default Card
