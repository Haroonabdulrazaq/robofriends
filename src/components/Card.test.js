import React from 'react';
import { shallow } from 'enzyme';
import Card from "./Card";

import {configure} from 'enzyme';

import Adapter from '@wojtekmaj/enzyme-adapter-react-17';




configure({ adapter: new Adapter() });


it('Wetin they occur',()=>{
    expect(shallow(<Card/>)).toMatchSnapshot() 
})



