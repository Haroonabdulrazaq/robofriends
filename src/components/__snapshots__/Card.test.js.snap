// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wetin they occur 1`] = `ShallowWrapper {}`;
